js/client-teambuilder.js

```diff
@@ -450,6 +450,7 @@
 		formatChange: function(e) {
 			this.curTeam.format = e.currentTarget.value;
 			this.save();
+			if (this.currTeam.format.substr(0, 4) === 'gen5' && !Tools.loadedSpriteData['bw']) Tools.loadSpriteData('bw');
```

Comment:
I apparently missed to test the actual condition here when I made sure I matched the correct substring.